profile.yml

```diff
@@ -0,0 +1,85 @@
+---
+# All variables needed to deploy the tripleo-quickstart-scalelab project.
+
+rhos_release_rpm: "{{ lookup('env', 'RHOS_RELEASE_URL') }}"
+
+# OSP/OSPd versioning and build
+version: 11
```

Comment:
These are the real build version details, do we just want to leave that for now?